polaris-react/src/components/Badge/tests/Badge.test.tsx

```diff
@@ -34,19 +34,18 @@ describe('<Badge />', () => {
   });
 
   it('does not add pip styles when progress is not provided', () => {
-    const badge = mountWithApp(<Badge status="attention" />);
-
-    expect(badge).not.toContainReactComponent('span', {
-      className: 'Pip',
+    const badge = mountWithApp(<Badge status="attention" />, {
+      features: {polarisSummerEditions2023: true},
```

Comment:
I do set it to true automatically but then I run tests twice, once with the default, and once with it forced to false. These would fail on the false run. Everything else I didn't explicitly pass this in for passes on both. Hope that makes sense!